### ISSUE TITLE
osx_defaults error

### ISSUE THREAD

**Issue by JPry** [ansible/ansible-modules-extras#754]
##### Issue Type:

Bug Report
##### Ansible Version:

1.9.2
##### Ansible Configuration:

Installed via Homebrew. No changes made.
##### Environment:

Running on OS X 10.10.4. Managing local system.
##### Summary:

I receive the following error when attempting to use the `osx_defaults` module:

```
ERROR: osx_defaults is not a legal parameter in an Ansible task or handler
```
##### Steps To Reproduce:

I've create a role named "osx". It has the following in the `defaults.yml` file, which is included from the role's `main.yml` file:

``` yaml

---
# OS X defaults

- name: Automatic scroll bars
  osx_defaults: key=AppleShowScrollBars value=Automatic
```

I have a playbook named `provision.yml` that runs that role:

``` yaml

---
- hosts: osx
  roles:
    - common
    - osx
```

Running the playbook with the following command yields the error:

```
ansible-playbook -i inventory/hosts provision.yml
```
##### Expected Results:

I expected the playbook to execute successfully.
##### Actual Results:

I received the following error:

```
ERROR: osx_defaults is not a legal parameter in an Ansible task or handler
```

I believe it is worth noting that in the same playbook, I have tasks which utilize `homebrew` and `homebrew-tap`. Those are also part of the Extras module, which leads me to believe the error is not due to the extras module not being found.

**Comment by bcoca** [comment]
As docs show: http://docs.ansible.com/ansible/osx_defaults_module.html it was added for version 2.0,  so it is not available for 1.9.2 which you are using.

**Comment by JPry** [comment]
@bcoca So sorry! I looked at that doc page so many times and completely missed the "new in 2.0" line. My mistake :smile:
